content/en/dashboards/widgets/scatter_plot.md

```diff
@@ -17,40 +17,32 @@ The scatter plot visualization allows you to graph a chosen scope over two diffe
 
 ## Setup
 
-{{< img src="dashboards/widgets/scatterplot/scatterplot_setup.png" alt="Scatter Plot Setup" style="width:80%;">}}
-
 ### Configuration
 
 1. Select a metric and an aggregation for the X and Y axis.
-2. Define the scope for each point of the scatter plot, such as `host`, `service`, `app`, `region`, etc.
-3. Optional: enable a color-by tag.
-4. Optional: set X and Y axis controls.
-
-## Options
-
-#### Global time
-
-On screenboards only, choose whether your widget has a custom timeframe or the screenboard's global timeframe.
-
-#### Title
-
-Display a custom title for your widget by activating the `Show a Title` check box:
+1. Define the scope for each point of the scatter plot, such as `host`, `service`, `app`, `region`, etc.
+1. Optional: enable a color-by tag.
+1. Optional: set X and Y axis controls.
+1. Choose whether your widget has a custom timeframe or the dashboard's global timeframe.
+1. Give your graph a title or leave the box blank for the suggested title.
 
-{{< img src="dashboards/widgets/options/title.png" alt="Widget title" style="width:80%;">}}
+### Options
 
-Optionally define its size and alignment.
+#### Context links
 
+[Context links][1] are enabled by default, and can be toggled on or off. Context links bridge dashboard widgets with other pages (in Datadog, or third-party).
```

Comment:
```suggestion
[Context links][1] are enabled by default, and can be toggled on or off. Context links bridge dashboard widgets with other pages in Datadog, or third-party applications.
```
Not sure about this one. You could also use `a third party` (no hyphen for noun form).